sites/bioopticsworld.com/server/templates/index.marko

```diff
@@ -23,7 +23,6 @@ $ const { id, alias, name, pageNode } = data;
   <@above-container>
     <marko-web-resolve-page|{ data: section }| node=pageNode>
       $ const aliases = hierarchyAliases(section);
-      <marko-web-gam-out-of-page-ad ...GAM.getAdUnit({ name: "wa", aliases }) />
```

Comment:
The welcome ad is present on more than just home page in some cases. You’ll want to double-check that it’s removed from all other pages